package.json

```diff
@@ -346,5 +346,5 @@
       "enzyme-to-json/serializer"
     ]
   },
-  "bundleSizeThreshold": 120000
+  "bundleSizeThreshold": 150000
```

Comment:
Raising the debt ceiling 😢

Reply:
Yeah I'm not sure what's happening with the size increase, it feels like our bundling is off somewhere and we'll have to start digging into why. My hope is that this upgrade PR will get merged and then we can start updating our bundle output from transpiled babel files to a single entrypoint

Reply:
@vpicone bumped it back down with the `module` change!